pubsub/azuresb/azuresb.go

```diff
@@ -476,6 +477,15 @@ func (s *subscription) ReceiveBatch(ctx context.Context, maxMessages int) ([]*dr
 			})
 			return nil
 		}))
+
+		if err != nil {
```

Comment:
Per the driver contract for ReceiveBatch (https://github.com/google/go-cloud/blob/master/pubsub/driver/driver.go#L124),

```
	// If there is a transient failure, this method should not retry but
	// should return a nil slice and an error. The concrete API will take
	// care of retry logic.
```